play.pokemonshowdown.com/src/panel-chat.tsx

```diff
@@ -314,6 +314,7 @@ export class ChatRoom extends PSRoom {
 		'reject'(target) {
 			this.challenged = null;
 			this.update(null);
+			this.sendDirect(`/reject ${this.pmTarget!}`);
```

Comment:
```suggestion
			this.sendDirect(`/reject ${this.pmTarget || ''}`);
```

`pmTarget` is not guaranteed to exist here.